### ISSUE TITLE
Fix find_unused for security groups associated with lambdas that aren't running

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/cloudmapper#486]
For #372, I used the technique that other tools use of just listing the security groups and listing the network interfaces and doing a diff.  This does not work for VPC Lambdas, which do not have network interfaces unless they are running.  So you can have Lambdas that maybe run nightly, that need security groups, but if you run this during the day, you would find their associated security groups are unused.

To account for this I can either:
1) Create the cloudmapper network and find unused groups that way.  The downside of this is it takes a long time to run, and potentially still will miss things that CloudMapper's graph does not account for.
2) Add in one-off functionality.  The downside of this is adding extra code.

**Comment by 0xdabbad00** [comment]
I added in the CloudMapper network code:
```
    # Get the data from the `prepare` command
    outputfilter = {
        "internal_edges": True,
        "read_replicas": True,
        "inter_rds_edges": True,
        "azs": False,
        "collapse_by_tag": None,
        "collapse_asgs": True,
        "mute": True
    }
    outputfilter["regions"] = '"{}"'.format(region.name)
    network = build_data_structure(
        {'id':region.account.local_id, 'name': region.account.name}, 
        {'cidrs':{}}, 
        outputfilter)
    
    for edge in pyjq.all('.[].data|select(.type=="edge")', network):
        for sg in edge.get("node_data", []):
            if type(sg) is not list:
                used_sgs.add(sg.get("GroupId", None))
```

But this missed my example test case, in which I randomly added a Security Group to a VPC Lambda that allows an external IP.  This didn't get marked as in use, because it didn't show up on an edge, because no communication can exist between a Lambda and an external IP.  So technically, this Security Group is "in-use" even though it can't actually do anything. This brings up two problems:
1. Do we want to mark a security group as being unused, if it is used but not useful?
2. There are cases that CloudMapper's network view isn't going to account for.  For example, our Lambda likely only is going to perform egress.  CloudMapper's network view does not take egress traffic into consideration, so this could miss that that Security Group is in-use. 

For these reasons, I think I have to go the one-off route, or I could leverage some fo the code that builds the nodes, and reads the Security Groups directly from those.

**Comment by 0xdabbad00** [comment]
Resolved in #489
